ci-operator/step-registry/baremetal/lab/post/wipe/baremetal-lab-post-wipe-commands.sh

```diff
@@ -54,6 +54,23 @@ function wait_for_power_down() {
   return 0
 }
 
+function reset_pdu() {
+  local pdu_uri="${1}"
+  pdu_host=${pdu_uri%%/*}
+  pdu_socket=${pdu_uri##*/}
+  pdu_creds=${pdu_host%%@*}
+  pdu_host=${pdu_host##*@}
+  pdu_user=${pdu_creds%%:*}
+  pdu_pass=${pdu_creds##*:}
+  # pub-priv key auth is not supported by the PDUs
+  echo "${pdu_pass}" > /tmp/ssh-pass
+
+  timeout -s 9 10m sshpass -f /tmp/ssh-pass ssh "${SSHOPTS[@]}" "${pdu_user}@${pdu_host}" <<EOF || true
```

Comment:
```suggestion
  timeout -s 9 1m sshpass -f /tmp/ssh-pass ssh "${SSHOPTS[@]}" "${pdu_user}@${pdu_host}" <<EOF || true
```

should we enough for this task, right?